src/unix/udp.c

```diff
@@ -349,7 +349,10 @@ int uv__udp_bind(uv_udp_t* handle,
 static int uv__udp_maybe_deferred_bind(uv_udp_t* handle,
                                        int domain,
                                        unsigned int flags) {
-  unsigned char taddr[sizeof(struct sockaddr_in6)];
+  union {
+    struct sockaddr_in6 in6;
+    struct sockaddr_in in;
+  } taddr;
```

Comment:
Can you add a `struct sockaddr` field and pass its address in the call to `uv__udp_bind()`?